server/bundler/extensions-loader.js

```diff
@@ -0,0 +1,34 @@
+const existsSync = require( 'fs' ).existsSync;
+const path = require( 'path' );
+const camelCase = require( 'lodash' ).camelCase;
+
+function getReducers( extensions = {} ) {
+	return extensions
+		.filter( ( extension ) => existsSync( path.join( 'client', 'extensions', extension, 'state', 'reducer.js' ) ) )
```

Comment:
Okay, Justin and I spent some time getting this in, but I think we've got it solved appropriately. Reviews definitely welcome with this new code here.